podcasts/PlaybackManager.swift

```diff
@@ -1352,6 +1352,7 @@ class PlaybackManager: ServerPlaybackDelegate {
                 } else {
                     FileLog.shared.addMessage("Skipping last \(timeRemaining) seconds of episode because podcast has skip last of \(skipLast) set.")
                     StatsManager.shared.addAutoSkipTime(timeRemaining)
+                    playerDidFinishPlayingEpisode()
                     EpisodeManager.markAsPlayed(episode: episode, fireNotification: true)
```

Comment:
@SergioEstevao Is the call to `EpisodeManager.markAsPlayed()` still needed? I think all the episode cleanup is already done via `playerDidFinishPlayingEpisode()`.

Also, this will fix the bug for Skip Outro, but not the similar bug when episode is marked as played from Mini Player or Episode list. It would be good to fix all instances, if possible.